OMCompiler/Compiler/BackEnd/BackendDAECreate.mo

```diff
@@ -173,7 +173,7 @@ algorithm
                                                     NONE()
                                                     ));
   BackendDAEUtil.checkBackendDAEWithErrorMsg(outBackendDAE);
-  BackendDAEUtil.checkIncidenceMatrixSolvability(syst, functionTree);
+  BackendDAEUtil.checkIncidenceMatrixSolvability(syst, functionTree,BackendDAEUtil.isInitializationDAE(outBackendDAE.shared));
```

Comment:
Please be consistent with space after each argument (Or no space at all)

Reply:
I tried. Whenever i change functions i try to replicate the surrounding style, but sometimes i fail. :D